components/channel_notifications_modal/channel_notifications_modal.jsx

```diff
@@ -247,5 +247,6 @@ ChannelNotificationsModal.propTypes = {
     onHide: PropTypes.func.isRequired,
     channel: PropTypes.object.isRequired,
     channelMember: PropTypes.object.isRequired,
-    currentUser: PropTypes.object.isRequired
+    currentUser: PropTypes.object.isRequired,
+    sendPushNotifications: PropTypes.bool.isRequired
```

Comment:
We can move the propTypes to the `static propType` class attribute. Not a stopper.